Source/Core/LMCHFSSResponseFileHandler.hh

```diff
@@ -73,14 +74,15 @@ namespace locust
         // Member functions
         virtual bool Configure( const scarab::param_node& aNode) override;
         bool ReadHFSSFile() override;
+	bool GenerateAnalyticTFtoFIR(double initialFreq, std::vector<std::complex<double>> tfArray) override;
```

Comment:
Would it make sense to call this `ConvertAnalyticTFtoFIR` ? 
You could also just move all the functionality you define in this function into an ` if(GeneratedTF)` loop inside the 
`ConvertTFtoFIR` and that way you can completely avoid having this function and just directly call `ConvertTFtoFIR` function.

Reply:
I agree that `ConvertAnalyticTFtoFIR` is more intuitive, so I have changed it.